examples/step-75/step-75.cc

```diff
@@ -1160,7 +1160,8 @@ namespace Step75
         Utilities::MPI::gather(mpi_communicator,
                                triangulation.n_locally_owned_active_cells());
       for (unsigned int i = 0; i < first_n_processes; ++i)
-        pcout << ' ' << n_active_cells_per_subdomain[i];
+        if (n_active_cells_per_subdomain.size() > 0)
+          pcout << ' ' << n_active_cells_per_subdomain[i];
```

Comment:
This is hard to read. It wasn't clear to me why the array would have zero size. The controlling condition is actually that `MPI::gather` only returns its value on process zero, but the code does not make that clear. I think it would be easier to read if you just called `allgather` above. That's perhaps not the most efficient, but I don't think it matters here, and it allows you to not have to add a line you'd need to explain.

Same below of course.

Reply:
Alternatively, you can add the necessary documentation to explain why you have that `if` condition.

Reply:
I have added a paragraph in the documentation.